src/main/java/org/chameleoncloud/api/admin/ChameleonAdminRoot.java

```diff
@@ -23,16 +21,12 @@ public ChameleonAdminRoot(KeycloakSession session) {
     }
 
     @Path("user-group-roles")
-    public UserGroupRolesResource userGroupRoles(@Context final HttpRequest request,
-                                                 @Context HttpResponse response) {
+    public UserGroupRolesResource userGroupRoles(@Context HttpHeaders headers) {
```

Comment:
The jboss dependencies changed a bunch, so nothing we imported there works anymore. I think this update will work, but needs better testing on dev.